PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,5 +7,6 @@ Risk Level:
 Testing:
 Docs Changes:
 Release Notes:
+[Optional Runtime guard:]
```

Comment:
Document more fully in https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md with links to the contributing guide, etc.?